install.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+set -u # error on use of undefined variable
+set -e # exit on error
+
+if command -v stack &> /dev/null ; then
+  echo "found stack"
+else
+  echo "Haskell stack not found."
+  echo "Installing Haskell stack to."
+  curl -sSL https://get.haskellstack.org/ | sh
+  if [[ -x "$install_path/stack" ]]; then
```

Comment:
`install_path` is used before being set in line 20.

Reply:
@awvwgk I just pushed a fix for this.  Thanks!